programmers/문자열-내림차순으로-배치하기.js

```diff
@@ -0,0 +1,13 @@
+//�� ��� Ǯ��*****
+// function solution(s) {
+//     return s
+//         .split("")
+//         .sort((a, b) => a > b ? -1 : 0)
+//         .join("");
+// }
+//�� �ߴ� Ǯ��*****
+function solution(s) {
+    let arr = s.split("");
+    arr.sort((a, b) => a > b ? -1 : 0);
+    return arr.join("");
+}
```

Comment:
```suggestion
function solution(s) {
    let arr = s.split("");
    arr.sort((a, b) => a > b ? -1 : 0);
    return arr.join("");
}
```
```js
function solution(s) {
    const arr = s.split("");
    return arr.sort((a, b) => a > b ? -1 : 0).join('');
}
```
```js
function solution(s) {
    return s.split('').sort((a, b) => a > b ? -1 : 0).join('');
}
```

이런 과정으로 코드양을 줄일 수 있군여
reverse대신 구현한 방법 상당히 훌륭하고, 
let을 사용할때는 const로는 안될라나? 라고 생각하고 사용하세요  

이 문제에 대한 내일 숙제는
```js
function solution(s) {
    return [...s].sort().reverse().join('')
 }
```
여기에서 [...s] 가 뭔지 알아오기입니다